projects/simple/tests/Tests/Example/Project.hs

```diff
@@ -1,26 +1,39 @@
 module Tests.Example.Project where
 
-import Prelude
+import Clash.Prelude
```

Comment:
Since if you remove the unnecessary `outputVerifier`, this is normal Haskell code, I think it's better if you import the Clash Prelude qualified and use the normal Prelude otherwise.